packages/contracts-bedrock/test/L1/L1CrossDomainMessenger.t.sol

```diff
@@ -22,6 +24,46 @@ contract L1CrossDomainMessenger_Test is Bridge_Initializer {
     /// @dev The storage slot of the l2Sender
     uint256 constant senderSlotIndex = 50;
 
+    /// @dev Tests that the proxy is initialized correctly.
+    function test_params_initValuesOnProxy_succeeds() external {
+        assertEq(address(l1CrossDomainMessenger.superchainConfig()), address(superchainConfig));
+        assertEq(address(l1CrossDomainMessenger.PORTAL()), address(optimismPortal));
+        assertEq(l1CrossDomainMessenger.portal(), address(optimismPortal));
+        assertEq(l1CrossDomainMessenger.OTHER_MESSENGER(), Predeploys.L2_CROSS_DOMAIN_MESSENGER);
+        assertEq(l1CrossDomainMessenger.otherMessenger(), Predeploys.L2_CROSS_DOMAIN_MESSENGER);
+    }
+
+    /// @dev Tests that the impl is initialized correctly.
+    function test_params_initValuesOnImpl_succeeds() external {
+        L1CrossDomainMessenger impl = L1CrossDomainMessenger(deploy.mustGetAddress("L1CrossDomainMessenger"));
+        assertEq(address(impl.superchainConfig()), address(0));
+        assertEq(address(impl.PORTAL()), address(0));
+        assertEq(impl.portal(), address(0));
+        assertEq(impl.OTHER_MESSENGER(), Predeploys.L2_CROSS_DOMAIN_MESSENGER);
+        assertEq(impl.otherMessenger(), Predeploys.L2_CROSS_DOMAIN_MESSENGER);
+    }
+
+    /// @dev Tests that the proxy cannot be initialized twice.
+    function test_initialize_cannotInitProxy_reverts() external {
+        vm.expectRevert("Initializable: contract is already initialized");
+        l1CrossDomainMessenger.initialize({
+            _superchainConfig: SuperchainConfig(address(0)),
+            _portal: payable(address(0)),
+            _l2CrossDomainMessenger: Predeploys.L2_CROSS_DOMAIN_MESSENGER
+        });
+    }
+
+    /// @dev Tests that the implementation cannot be initialized twice.
+    function test_initialize_cannotInitImpl_reverts() external {
```

Comment:
This test should be covered in the `Initializable.t.sol`

Reply:
Can you be sure to update that file for the modified contracts in this PR?

Reply:
yes absolutely!